### DISCUSSION TITLE
ERROR failed to auto-install CurseForge modpack

### DISCUSSION THREAD

**Discussion by GarretCMK** [itzg/docker-minecraft-server#2957] [Q&A]
### Describe the problem

Had the server running 1 hour ago and reset and now it's all messed up. These mods are in the folder with correct permissions.
ls -al of mods folder...
> drwxr-xr-x  2 garretcmk garretcmk    12288 Mar 14 20:58  .
> drwxr-xr-x 19 garretcmk garretcmk     4096 Mar 15 03:38  ..
> -rw-r--r--  1 garretcmk garretcmk   115538 Jan 21 12:10  AE2-Things-1.0.5.jar
> -rw-r--r--  1 garretcmk garretcmk    42265 Jan 21 12:10  AkashicTome-1.5-20.jar
> -rw-r--r--  1 garretcmk garretcmk 22436427 Jan 21 12:10  alexsmobs-1.18.6.jar
> -rw-r--r--  1 garretcmk garretcmk    38226 Feb  9 06:23  alternate-current-mc1.18-1.7.0.jar
> -rw-r--r--  1 garretcmk garretcmk    46304 Jan 21 12:10  appleskin-forge-mc1.18.2-2.4.1.jar
> -rw-r--r--  1 garretcmk garretcmk  5126390 Jan 21 12:10  appliedenergistics2-forge-11.7.0.jar
> -rw-r--r--  1 garretcmk garretcmk  2131192 Jan 21 12:10  Architects-Palette-1.18.2-1.3.2.jar
> -rw-r--r--  1 garretcmk garretcmk   548109 Jan 21 12:10  architectury-4.10.88-forge.jar
> -rw-r--r--  1 garretcmk garretcmk    42733 Jan 21 12:10  AutoRegLib-1.7-53.jar
> -rw-r--r--  1 garretcmk garretcmk    27697 Jan 21 12:10  auudio_forge_1.0.3_MC_1.18-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   872653 Jan 21 12:10  auxiliaryblocks-1.18.2-0.4.1.jar
> -rw-r--r--  1 garretcmk garretcmk   621988 Jan 21 12:10  balm-3.2.1+0.jar
> -rw-r--r--  1 garretcmk garretcmk  1255404 Jan 21 12:10  bdlib-1.19.3.7-mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk    15653 Jan 21 12:10  BetterCompatibilityChecker-1.1.21-build.48+mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   723006 Jan 21 12:10  blockcarpentry-1.18-0.3.0.jar
> -rw-r--r--  1 garretcmk garretcmk   312261 Jan 21 12:10  Bookshelf-Forge-1.18.2-13.2.51.jar
> -rw-r--r--  1 garretcmk garretcmk 11422451 Jan 21 12:10  Botania-1.18.2-435.jar
> -rw-r--r--  1 garretcmk garretcmk   750306 Jan 21 12:10  BotanyPots-Forge-1.18.2-8.0.23.jar
> -rw-r--r--  1 garretcmk garretcmk   895524 Jan 21 12:10  buildinggadgets-3.13.2-build.21+mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk    99383 Jan 21 12:10  cagerium-1.18.2-1.1.6.jar
> -rw-r--r--  1 garretcmk garretcmk   291141 Jan 21 12:10  camera-1.18.2-1.0.5.jar
> -rw-r--r--  1 garretcmk garretcmk  2181196 Jan 21 12:10  cfm-7.0.0-pre35-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk  6166031 Jan 21 12:10  chipped-1.18.2-1.2.1-forge.jar
> -rw-r--r--  1 garretcmk garretcmk    68785 Jan 21 12:10  ChunkyMcChunkFace-1.18.2-0.3.2.jar
> -rw-r--r--  1 garretcmk garretcmk   503137 Jan 21 12:10  citadel-1.11.3-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk  1136441 Jan 21 12:10  cloth-config-6.3.81-forge.jar
> -rw-r--r--  1 garretcmk garretcmk    19070 Jan 21 12:10  Clumps-forge-1.18.2-8.0.0+17.jar
> -rw-r--r--  1 garretcmk garretcmk  1405426 Jan 21 12:10  cofh_core-1.18.2-9.1.2.32.jar
> -rw-r--r--  1 garretcmk garretcmk   225123 Jan 21 12:10  ColossalChests-1.18.2-1.8.3.jar
> -rw-r--r--  1 garretcmk garretcmk   225616 Jan 21 12:10  comforts-forge-1.18.2-5.0.0.5.jar
> -rw-r--r--  1 garretcmk garretcmk   673208 Jan 21 12:10  Compressium-1.4.2-build.9+mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   683519 Jan 21 12:10  cookingforblockheads-forge-1.18.2-12.2.0.jar
> -rw-r--r--  1 garretcmk garretcmk   219786 Jan 21 12:10  corpse-1.18.2-1.0.2.jar
> -rw-r--r--  1 garretcmk garretcmk   178181 Jan 21 12:10  craftingtweaks-forge-1.18.2-14.0.3.jar
> -rw-r--r--  1 garretcmk garretcmk  2925616 Jan 21 12:10  CraftTweaker-forge-1.18.2-9.1.207.jar
> -rw-r--r--  1 garretcmk garretcmk 14596095 Jan 21 12:10  create-1.18.2-0.5.1.b.jar
> -rw-r--r--  1 garretcmk garretcmk   106237 Jan 21 12:10  CreateTweaker-1.18.2-2.0.0.19.jar
> -rw-r--r--  1 garretcmk garretcmk   167776 Jan 21 12:10  Cucumber-1.18.2-5.1.3.jar
> -rw-r--r--  1 garretcmk garretcmk   276862 Jan 21 12:10  curios-forge-1.18.2-5.0.9.0.jar
> -rw-r--r--  1 garretcmk garretcmk   870122 Jan 21 12:10  CyclopsCore-1.18.2-1.16.1.jar
> -rw-r--r--  1 garretcmk garretcmk   164318 Jan 21 12:10  DarkUtilities-Forge-1.18.2-10.0.5.jar
> -rw-r--r--  1 garretcmk garretcmk   468604 Jan 21 12:10 'Decorative Blocks-forge-1.18.2-2.1.2.jar'
> -rw-r--r--  1 garretcmk garretcmk   403520 Jan 21 12:10  drippyloadingscreen_forge_1.6.6_MC_1.18-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   184400 Jan 21 12:10  easy_piglins-1.18.2-1.0.4.jar
> -rw-r--r--  1 garretcmk garretcmk   364712 Jan 21 12:10  easy_villagers-1.18.2-1.0.11.jar
> -rw-r--r--  1 garretcmk garretcmk  1776460 Jan 21 12:10  ecologics-forge-1.18.2-1.7.11.jar
> -rw-r--r--  1 garretcmk garretcmk   185669 Jan 21 12:10  elevatorid-1.18.2-1.8.4.jar
> -rw-r--r--  1 garretcmk garretcmk    80487 Jan 21 12:10  enercell-1.18.2-1.0.2.jar
> -rw-r--r--  1 garretcmk garretcmk  1360675 Jan 21 12:10  everycomp-1.18.2-1.5.18.jar
> -rw-r--r--  1 garretcmk garretcmk   810431 Jan 21 12:10  fairylights-5.0.0-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk    22921 Jan 21 12:10  FastWorkbench-1.18.2-6.0.2.jar
> -rw-r--r--  1 garretcmk garretcmk   116626 Jan 21 12:10  ferritecore-4.2.2-forge.jar
> -rw-r--r--  1 garretcmk garretcmk   503606 Jan 21 12:10  FluxNetworks-1.18.2-7.0.8.12.jar
> -rw-r--r--  1 garretcmk garretcmk   124741 Jan 21 12:10  framedcompactdrawers-1.18-4.1.0.jar
> -rw-r--r--  1 garretcmk garretcmk  3740584 Jan 21 12:10  ftbbackups2-forge-1.18.2-1.0.19.jar
> -rw-r--r--  1 garretcmk garretcmk  2927832 Jan 21 12:10  geckolib-forge-1.18-3.0.57.jar
> -rw-r--r--  1 garretcmk garretcmk    24803 Jan 21 12:10  Highlighter-1.18.1-1.1.2.jar
> -rw-r--r--  1 garretcmk garretcmk   107482 Jan 21 12:10  Iceberg-1.18.2-forge-1.0.49.jar
> -rw-r--r--  1 garretcmk garretcmk    28978 Jan 21 12:10  InitialInventory-forge-1.18.2-6.0.8.jar
> -rw-r--r--  1 garretcmk garretcmk   601584 Jan 21 12:10  ironfurnaces-1.18.2-3.3.3.jar
> -rw-r--r--  1 garretcmk garretcmk   673657 Jan 21 12:10  IronGenerators-2.0.2-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   189363 Feb  5 00:42  ispawner-1.1.8-forge.jar
> -rw-r--r--  1 garretcmk garretcmk   298590 Jan 21 12:10  Jade-1.18.2-forge-5.2.6.jar
> -rw-r--r--  1 garretcmk garretcmk   999657 Jan 21 12:10  jei-1.18.2-9.7.2.1001.jar
> -rw-r--r--  1 garretcmk garretcmk   156552 Jan 21 12:10  JEITweaker-1.18.2-3.0.0.9.jar
> -rw-r--r--  1 garretcmk garretcmk   641583 Jan 21 12:10  konkrete_forge_1.6.0_MC_1.18-1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk    15445 Feb  9 06:23  lazydfu-1.0-1.18+.jar
> -rw-r--r--  1 garretcmk garretcmk   299918 Jan 21 12:10  libnonymous-2.1.0.jar
> -rw-r--r--  1 garretcmk garretcmk    12324 Feb  9 06:26  logprot-1.18.2-1.7.jar
> -rw-r--r--  1 garretcmk garretcmk    61835 Jan 21 12:10  lootbeams-1.18.1-release-july1722.jar
> -rw-r--r--  1 garretcmk garretcmk   956674 Jan 21 12:10  Mantle-1.18.2-1.9.31.jar
> -rw-r--r--  1 garretcmk garretcmk   651087 Jan 21 12:10  mcjtylib-1.18-6.0.20.jar
> -rw-r--r--  1 garretcmk garretcmk   569260 Jan 21 12:10  mcw-bridges-2.0.6-mc1.18.2forge.jar
> -rw-r--r--  1 garretcmk garretcmk   798780 Jan 21 12:10  mcw-doors-1.0.7-mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   317432 Jan 21 12:10  mcw-fences-1.0.7-mc1.18.2forge.jar
> -rw-r--r--  1 garretcmk garretcmk  1449569 Jan 21 12:10  mcw-furniture-3.0.2-mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   242271 Jan 21 12:10  mcw-lights-1.0.4-mc1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   489091 Jan 21 12:10  mcw-trapdoors-1.0.8-mc1.18.2forge.jar
> -rw-r--r--  1 garretcmk garretcmk   776948 Jan 21 12:10  mcw-windows-2.1.1-mc1.18.2forge.jar
> -rw-r--r--  1 garretcmk garretcmk 11619466 Jan 21 12:10  Mekanism-1.18.2-10.2.5.465.jar
> -rw-r--r--  1 garretcmk garretcmk   991220 Jan 21 12:10  MekanismGenerators-1.18.2-10.2.5.465.jar
> -rw-r--r--  1 garretcmk garretcmk   169748 Jan 21 12:10  merequester-1.18.2-1.1.2.jar
> -rw-r--r--  1 garretcmk garretcmk   609979 Jan 21 12:10  mininggadgets-1.11.1.jar
> -rw-r--r--  1 garretcmk garretcmk   134770 Jan 21 12:10  MmmMmmMmmMmm-1.18.2-1.5.2.jar
> -rw-r--r--  1 garretcmk garretcmk  1104671 Jan 21 12:10  modular-routers-1.18.2-9.1.1-93.jar
> -rw-r--r--  1 garretcmk garretcmk   206034 Jan 21 12:10  NaturesCompass-1.18.2-1.9.7-forge.jar
> -rw-r--r--  1 garretcmk garretcmk   800224 Jul 19  2023  neoncraft2-2.2.jar
> -rw-r--r--  1 garretcmk garretcmk    21632 Jan 21 12:10  OpenLoader-Forge-1.18.2-12.0.1.jar
> -rw-r--r--  1 garretcmk garretcmk   542906 Jan 21 12:10  PackMenu-1.18.2-4.1.3.jar
> -rw-r--r--  1 garretcmk garretcmk   631415 Jan 21 12:10  Patchouli-1.18.2-71.1.jar
> -rw-r--r--  1 garretcmk garretcmk    81610 Jan 21 12:10  PickUpNotifier-v3.2.1-1.18.2-Forge.jar
> -rw-r--r--  1 garretcmk garretcmk    10260 Jan 21 12:10  PigPen-Forge-1.18.2-8.0.1.jar
> -rw-r--r--  1 garretcmk garretcmk    77693 Jan 21 12:10  Ping-1.18-1.8.0.jar
> -rw-r--r--  1 garretcmk garretcmk   402903 Jan 21 12:10  pipez-1.18.2-1.1.5.jar
> -rw-r--r--  1 garretcmk garretcmk   178703 Jan 21 12:10  Placebo-1.18.2-6.2.0.jar
> -rw-r--r--  1 garretcmk garretcmk   133490 Jan 21 12:10  polylib-forge-1801.0.2-build.13.jar
> -rw-r--r--  1 garretcmk garretcmk   259839 Jan 21 12:10  polymorph-forge-1.18.2-0.49.jar
> -rw-r--r--  1 garretcmk garretcmk  1900614 Jan 21 12:10  Powah-3.0.7.jar
> -rw-r--r--  1 garretcmk garretcmk    51513 Jan 21 12:10  Prism-1.18.2-1.0.1.jar
> -rw-r--r--  1 garretcmk garretcmk   127173 Jan 21 12:10  PuzzlesLib-v3.3.5-1.18.2-Forge.jar
> -rw-r--r--  1 garretcmk garretcmk 13743162 Jan 21 12:10  Quark-3.2-358.jar
> -rw-r--r--  1 garretcmk garretcmk   501920 Feb  9 06:22  radium-0.7.10.jar
> -rw-r--r--  1 garretcmk garretcmk  3252284 Feb  5 00:42  refinedstorage-1.10.6.jar
> -rw-r--r--  1 garretcmk garretcmk    32396 Jan 21 12:10  refinedstorageaddons-0.8.2.jar
> -rw-r--r--  1 garretcmk garretcmk   412506 Jan 21 12:10  rftoolsbase-1.18-3.0.11.jar
> -rw-r--r--  1 garretcmk garretcmk   391730 Jan 21 12:10  rottencreatures-forge-1.18.2-1.0.0.jar
> -rw-r--r--  1 garretcmk garretcmk    61737 Jan 21 12:10  rsrequestify-2.2.0.jar
> -rw-r--r--  1 garretcmk garretcmk    14146 Jan 21 12:10  Runelic-Forge-1.18.2-11.0.1.jar
> -rw-r--r--  1 garretcmk garretcmk    76625 Feb  9 06:23  saturn-mc1.18.2-0.1.3.jar
> -rw-r--r--  1 garretcmk garretcmk 15738742 Jan 21 12:10  ScalableCatsForce-2.13.10-build-12-with-library.jar
> -rw-r--r--  1 garretcmk garretcmk   456163 Jan 21 12:10  selene-1.18.2-1.17.9.jar
> -rw-r--r--  1 garretcmk garretcmk    13660 Jan 21 12:10  serverconfigupdater-2.2.jar
> -rw-r--r--  1 garretcmk garretcmk   456292 Jan 21 12:10  shetiphiancore-1.18-3.10.12.jar
> -rw-r--r--  1 garretcmk garretcmk    40305 Jan 21 12:10  shulkerplus-mc1.18.2-v1.0.7-forge.jar
> -rw-r--r--  1 garretcmk garretcmk     4748 Jan 21 12:10  shutupexperimentalsettings-1.0.5.jar
> -rw-r--r--  1 garretcmk garretcmk   461698 Jan 21 12:10  SimpleStorageNetwork-1.18.2-1.6.2.jar
> -rw-r--r--  1 garretcmk garretcmk   549014 Jan 21 12:10  simplylight-1.18.2-1.4.5-build.43.jar
> -rw-r--r--  1 garretcmk garretcmk    23603 Jan 21 12:10  Snad-1.18.2-1.22.04.15a.jar
> -rw-r--r--  1 garretcmk garretcmk   776078 Jan 21 12:10  sophisticatedbackpacks-1.18.2-3.18.46.821.jar
> -rw-r--r--  1 garretcmk garretcmk    66936 Jan 21 12:10  sophisticatedbackpacksvh-1.18.2-1.0.5.13.jar
> -rw-r--r--  1 garretcmk garretcmk  1000684 Jan 21 12:10  sophisticatedcore-1.18.2-0.5.48.244.jar
> -rw-r--r--  1 garretcmk garretcmk  2976453 Jan 21 12:10  spark-1.10.38-forge.jar
> -rw-r--r--  1 garretcmk garretcmk   110596 Feb  9 06:23  starlight-1.0.2+forge.546ae87.jar
> -rw-r--r--  1 garretcmk garretcmk   578059 Jan 21 12:10  StorageDrawers-1.18.2-10.2.1.jar
> -rw-r--r--  1 garretcmk garretcmk   205931 Jan 21 12:10  supermartijn642configlib-1.1.6-forge-mc1.18.jar
> -rw-r--r--  1 garretcmk garretcmk   206554 Jan 21 12:10  supermartijn642corelib-1.0.19-forge-mc1.18.jar
> -rw-r--r--  1 garretcmk garretcmk 13904989 Jan 21 12:10  supplementaries-1.18.2-1.5.18.jar
> -rw-r--r--  1 garretcmk garretcmk    96811 Jan 21 12:10  TerraBlender-forge-1.18.2-1.2.0.126.jar
> -rw-r--r--  1 garretcmk garretcmk  4497439 Jan 21 12:10  Terralith_v2.2.3.jar
> -rw-r--r--  1 garretcmk garretcmk   617207 Jan 21 12:10  thermal_expansion-1.18.2-9.1.0.18.jar
> -rw-r--r--  1 garretcmk garretcmk  4865862 Jan 21 12:10  thermal_foundation-1.18.2-9.1.0.34.jar
> -rw-r--r--  1 garretcmk garretcmk 28868020 Feb  5 00:43  the_vault-1.18.2-3.11.0.3011.jar
> -rw-r--r--  1 garretcmk garretcmk   595548 Jan 21 12:10  titanium-1.18.2-3.5.8-40.jar
> -rw-r--r--  1 garretcmk garretcmk    98651 Jan 21 12:10  torchmaster-18.1.0.jar
> -rw-r--r--  1 garretcmk garretcmk   148996 Jan 21 12:10  trashcans-1.0.15-forge-mc1.18.jar
> -rw-r--r--  1 garretcmk garretcmk   817165 Jan 21 12:10  twigs-1.1.4-patch4+1.18.2-forge.jar
> -rw-r--r--  1 garretcmk garretcmk    79482 Jan 21 12:11  vault_hunters_jewel_sorting-2.4.0.jar
> -rw-r--r--  1 garretcmk garretcmk    90004 Jan 21 12:10  vaultintegrations-1.18.2-1.0.13.jar
> -rw-r--r--  1 garretcmk garretcmk   247905 Jan 21 12:10  Waddles-1.18.2-0.8.19.jar
> -rw-r--r--  1 garretcmk garretcmk   357620 Jan 21 12:10  waystones-forge-1.18.2-10.1.0.jar
> -rw-r--r--  1 garretcmk garretcmk    71084 Jan 21 12:10  weirdinggadget-1.18.1-2.2.11.jar
> -rw-r--r--  1 garretcmk garretcmk  1257010 Jan 21 12:10  Xaeros_Minimap_22.17.0_Forge_1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   708804 Jan 21 12:10  XaerosWorldMap_1.28.7_Forge_1.18.2.jar
> -rw-r--r--  1 garretcmk garretcmk   527546 Jan 21 12:10  xnet-1.18-4.0.7.jar

and i've also noticed it is not making a MOD_NEED_DOWNLOAD.txt not really sure why.
My minecraft server folder is at /minecraft so im pretty positive my mapping is correct, I had this working for the last 3 months and on a restart everything broke.

### Container definition

version: "3.8"

services:
  minecraft:
    image: itzg/minecraft-server
    container_name: minecraft
    tty: true
    stdin_open: true
    ports:
      - "25565:25565"
    environment:
      TYPE: AUTO_CURSEFORGE
      CF_API_KEY: HIDDEN
      CF_PAGE_URL: https://www.curseforge.com/minecraft/modpacks/vault-hunters-1-18-2
      MEMORY: "14G"
      VERSION: "1.18.2"
      EULA: "TRUE"
    volumes:
      # attach the relative directory 'data' to the container's /data path
      - /minecraft:/data

### Container logs

[mc-image-helper] 04:25:18.559 INFO  : Downloaded mod file mods/refinedstorage-1                                                                                         .10.6.jar
[mc-image-helper] 04:25:18.560 INFO  : Downloaded mod file mods/Highlighter-1.18                                                                                         .1-1.1.2.jar
[mc-image-helper] 04:25:18.565 INFO  : Downloaded mod file mods/TerraBlender-for                                                                                         ge-1.18.2-1.2.0.126.jar
[mc-image-helper] 04:25:18.571 INFO  : Downloaded mod file mods/buildinggadgets-                                                                                         3.13.2-build.21+mc1.18.2.jar
[mc-image-helper] 04:25:18.604 INFO  : Downloaded mod file mods/drippyloadingscr                                                                                         een_forge_1.6.6_MC_1.18-1.18.2.jar
[mc-image-helper] 04:25:18.654 INFO  : Downloaded mod file mods/mcw-lights-1.0.4                                                                                         -mc1.18.2.jar
[mc-image-helper] 04:25:18.661 INFO  : Downloaded mod file mods/PickUpNotifier-v                                                                                         3.2.1-1.18.2-Forge.jar
[mc-image-helper] 04:25:18.661 INFO  : Downloaded mod file mods/torchmaster-18.1                                                                                         .0.jar
[mc-image-helper] 04:25:18.782 INFO  : Downloaded mod file mods/CreateTweaker-1.                                                                                         18.2-2.0.0.19.jar
[mc-image-helper] 04:25:18.876 INFO  : Downloaded mod file mods/geckolib-forge-1                                                                                         .18-3.0.57.jar
[mc-image-helper] 04:25:18.879 INFO  : Downloaded mod file mods/Quark-3.2-358.ja                                                                                         r
[mc-image-helper] 04:25:18.885 INFO  : Downloaded mod file mods/CraftTweaker-for                                                                                         ge-1.18.2-9.1.207.jar
[mc-image-helper] 04:25:18.895 INFO  : Downloaded mod file mods/mcw-windows-2.1.                                                                                         1-mc1.18.2forge.jar
[mc-image-helper] 04:25:18.895 INFO  : Downloaded mod file mods/supermartijn642c                                                                                         orelib-1.0.19-forge-mc1.18.jar
[mc-image-helper] 04:25:18.915 INFO  : Downloaded mod file mods/Mekanism-1.18.2-                                                                                         10.2.5.465.jar
[mc-image-helper] 04:25:18.922 INFO  : Downloaded mod file mods/BotanyPots-Forge                                                                                         -1.18.2-8.0.23.jar
[mc-image-helper] 04:25:18.933 INFO  : Downloaded mod file mods/SimpleStorageNet                                                                                         work-1.18.2-1.6.2.jar
[mc-image-helper] 04:25:18.938 INFO  : Downloaded mod file mods/weirdinggadget-1                                                                                         .18.1-2.2.11.jar
[mc-image-helper] 04:25:18.943 INFO  : Downloaded mod file mods/jei-1.18.2-9.7.2                                                                                         .1001.jar
[mc-image-helper] 04:25:18.956 INFO  : Downloaded mod file mods/shulkerplus-mc1.                                                                                         18.2-v1.0.7-forge.jar
[mc-image-helper] 04:25:18.979 INFO  : Downloaded mod file mods/BetterCompatibil                                                                                         ityChecker-1.1.21-build.48+mc1.18.2.jar
[mc-image-helper] 04:25:18.984 INFO  : Downloaded mod file mods/polymorph-forge-                                                                                         1.18.2-0.49.jar
[mc-image-helper] 04:25:19.010 INFO  : Downloaded mod file mods/ChunkyMcChunkFac                                                                                         e-1.18.2-0.3.2.jar
[mc-image-helper] 04:25:19.054 INFO  : Downloaded mod file mods/DarkUtilities-Fo                                                                                         rge-1.18.2-10.0.5.jar
[mc-image-helper] 04:25:19.082 INFO  : Downloaded mod file mods/Placebo-1.18.2-6                                                                                         .2.0.jar
[mc-image-helper] 04:25:19.098 INFO  : Downloaded mod file mods/ispawner-1.1.8-f                                                                                         orge.jar
[mc-image-helper] 04:25:19.109 INFO  : Downloaded mod file mods/simplylight-1.18                                                                                         .2-1.4.5-build.43.jar
[mc-image-helper] 04:25:19.193 INFO  : Downloaded mod file mods/mcw-fences-1.0.7                                                                                         -mc1.18.2forge.jar
[mc-image-helper] 04:25:19.194 INFO  : Downloaded mod file mods/auudio_forge_1.0                                                                                         .3_MC_1.18-1.18.2.jar
[mc-image-helper] 04:25:19.283 INFO  : Downloaded mod file mods/AutoRegLib-1.7-5                                                                                         3.jar
[mc-image-helper] 04:25:19.289 INFO  : Downloaded mod file mods/Ping-1.18-1.8.0.                                                                                         jar
[mc-image-helper] 04:25:19.290 INFO  : Downloaded mod file mods/Xaeros_Minimap_2                                                                                         2.17.0_Forge_1.18.2.jar
[mc-image-helper] 04:25:19.293 INFO  : Downloaded mod file mods/Decorative Block                                                                                         s-forge-1.18.2-2.1.2.jar
[mc-image-helper] 04:25:19.294 INFO  : Downloaded mod file mods/waystones-forge-                                                                                         1.18.2-10.1.0.jar
[mc-image-helper] 04:25:19.399 INFO  : Downloaded mod file mods/sophisticatedbac                                                                                         kpacksvh-1.18.2-1.0.5.13.jar
[mc-image-helper] 04:25:19.416 INFO  : Downloaded mod file mods/cloth-config-6.3                                                                                         .81-forge.jar
[mc-image-helper] 04:25:19.423 INFO  : Downloaded mod file mods/modular-routers-                                                                                         1.18.2-9.1.1-93.jar
[mc-image-helper] 04:25:19.451 INFO  : Downloaded mod file mods/refinedstoragead                                                                                         dons-0.8.2.jar
[mc-image-helper] 04:25:19.520 INFO  : Downloaded mod file mods/trashcans-1.0.15                                                                                         -forge-mc1.18.jar
[mc-image-helper] 04:25:19.532 INFO  : Downloaded mod file mods/xnet-1.18-4.0.7.                                                                                         jar
[mc-image-helper] 04:25:19.539 INFO  : Downloaded mod file mods/polylib-forge-18                                                                                         01.0.2-build.13.jar
[mc-image-helper] 04:25:19.547 INFO  : Downloaded mod file mods/titanium-1.18.2-                                                                                         3.5.8-40.jar
[mc-image-helper] 04:25:19.744 INFO  : Downloaded mod file mods/Mantle-1.18.2-1.                                                                                         9.31.jar
[mc-image-helper] 04:25:19.747 INFO  : Downloaded mod file mods/CyclopsCore-1.18                                                                                         .2-1.16.1.jar
[mc-image-helper] 04:25:19.753 INFO  : Downloaded mod file mods/fairylights-5.0.                                                                                         0-1.18.2.jar
[mc-image-helper] 04:25:19.764 INFO  : Downloaded mod file mods/konkrete_forge_1                                                                                         .6.0_MC_1.18-1.18.2.jar
[mc-image-helper] 04:25:19.776 INFO  : Downloaded mod file mods/Botania-1.18.2-4                                                                                         35.jar
[mc-image-helper] 04:25:19.888 INFO  : Downloaded mod file mods/ftbbackups2-forg                                                                                         e-1.18.2-1.0.19.jar
[mc-image-helper] 04:25:20.550 INFO  : Downloaded mod file mods/ScalableCatsForc                                                                                         e-2.13.10-build-12-with-library.jar
[mc-image-helper] 04:25:20.583 INFO  : Downloaded mod file mods/chipped-1.18.2-1                                                                                         .2.1-forge.jar
[mc-image-helper] 04:25:20.630 INFO  : Downloaded mod file mods/supplementaries-                                                                                         1.18.2-1.5.18.jar
[mc-image-helper] 04:25:20.841 INFO  : Downloaded mod file mods/alexsmobs-1.18.6                                                                                         .jar
[mc-image-helper] 04:25:21.013 INFO  : Downloaded mod file mods/Terralith_v2.2.3                                                                                         .jar
[mc-image-helper] 04:25:21.368 INFO  : Downloaded mod file mods/the_vault-1.18.2-3.11.0.3011.jar
[mc-image-helper] 04:25:21.795 INFO  : Downloaded mod file mods/create-1.18.2-0.5.1.b.jar
[mc-image-helper] 04:25:22.851 INFO  : Downloading Forge installer 40.2.9 for Minecraft 1.18.2
[mc-image-helper] 04:25:23.359 INFO  : Running Forge 40.2.9 installer for Minecraft 1.18.2. This might take a while...

Some mod authors disallow automated downloads.
The following need to be manually downloaded into the repo or excluded:
(Also written to ./MODS_NEED_DOWNLOAD.txt)

Mod                      Version Name                       Filename                               Download page                                                         
=======================  =================================  =====================================  =========================================================================================
Vault Hunters Jewel ...  Vault Hunters Jewel Sorting 2.4.0  vault_hunters_jewel_sorting-2.4.0.jar  https://www.curseforge.com/minecraft/mc-mods/vault-hunters-jewel-sorting/download/4871298
Neon Craft 2 Mod         neoncraft2-2.2                     neoncraft2-2.2.jar                     https://www.curseforge.com/minecraft/mc-mods/neon-craft-2-mod/download/3726051
[init] ERROR failed to auto-install CurseForge modpack

**Comment by GarretCMK** [comment]
Also when running the find command on my whole system for MODS_NEED_DOWNLOAD.txt there are no results.
and i've tried CHOWN -R for root and re-running the compose and that still did not work.

**Comment by GarretCMK** [comment]
Update:
I reinstalled docker and docker compose, I have now found a MODS_NEED_DOWNLOAD.txt but it is not in the right spot which i assigned in the compose file... not sure whats going on
./var/snap/docker/common/var-lib-docker/volumes/docker-compose_minecraft/_data/MODS_NEED_DOWNLOAD.txt

**Comment by itzg** [comment]
The modpack probably got upgraded and new mods were required.

**Comment by rhullah** [comment]
Yeah, there's newer mods that need manual downloading. When you reinstalled everything, are you using the same compose file? Cause it doesn't look like `/minecraft:/data` is mounting.

**Comment by rhullah** [comment]
Another thing @GarretCMK, for Vault Hunters v3 specifically you'll need to [enable flight and command blocks](https://github.com/itzg/docker-minecraft-server/blob/master/examples/vault-hunters-1-18-2/docker-compose.yml).
